### ISSUE TITLE
Installation ubuntu vivid

### ISSUE THREAD

**Issue by ledshark24** [Rolisteam/rolisteam#4]
hi, 

I encounter issues to install (or compile) rolisteam on ubuntu 15.04 amd64 so far that I can't install it 

The ppa indicated in the documentation ends up to a 404 error code : `http://ppa.launchpad.net/rolisteam/ppa/ubuntu/dists/vivid/main/binary-amd64/Packages  404  Not Found`

so I tried to compile the latest tar.gz (rolisteam-1.6.1.tar.gz)  using instruction I found here : http://wiki.rolisteam.org/index.php/CompilationLinux 
it seems that prerequisites are out of date : 
`$ sudo apt-get install libqt4-core libqt4-gui libqt4-dev phonon phonon-backend-gstreamer libphonon4 libphonon-dev zlib1g-dev build-essential`
`$lrelease rolisteam.pro `
`lrelease: could not exec '/usr/lib/x86_64-linux-gnu/qt5/bin/lrelease': No such file or directory`
fixed using ` sudo apt-get install qttools5-dev-tools`
but then I got this : 
`qmake -r`
`Project ERROR: Unknown module(s) in QT: phonon`
I tried to install qt5 equivalents for phonon, it did not solve the problem.

Same results with a fresh clone of the repo.

I must do something wrong, but what ?

**Comment by obiwankennedy** [comment]
Yes, the packaging is only available in trusty.

So, you have to compile it yourself.
I change a bit the documentation to add in the install command another package: qt4-dev-tools

so try:
sudo apt-get install qt4-dev-tools

and rerun the compilation rules.
However, it looks like you have Qt5 installed and Qt4, that can cause some troubles.
When you start lrelease, be sure it is the qt4 version (this advice is also right for all commands).
As it is said on http://wiki.rolisteam.org/index.php/CompilationLinux#Having_trouble_with_Qt4_and_Qt5

Run:
lrelease-qt4 rolisteam.pro
qmake-qt4 -r rolisteam.pro
make

You won't be able to compile Rolisteam v1.6.1 if you compile with Qt5.
The head revision of Rolisteam in git works fine with qt5. It will be release soon.

Let me know if it helped you.
Regards.

**Comment by ledshark24** [comment]
Hi,

I was trying to compile it with qt5 that explains it... 
So following your instructions I only had the following error :  

> g++ -c -fstack-protector -W -Wall -Wextra -pedantic -Wstack-protector -Wno-long-long -Werror -O2 -w  
>-D_REENTRANT -DPHONON -DVERSION_MAJOR=1 -DVERSION_MIDDLE=6 
>-DVERSION_MINOR=1 -DQT_NO_DEBUG -DQT_PHONON_LIB -DQT_XML_LIB -DQT_GUI_LIB 
>-DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/usr/share/qt4/mkspecs/linux-g++-64 -I. -
>I/usr/include/qt4/QtCore -I/usr/include/qt4/QtNetwork -I/usr/include/qt4/QtGui -I/usr/include/qt4/QtXml 
>-I/usr/include/phonon -I/usr/include/qt4 -I. -Isrc -Isrc/odt/format_OO_oasis_ -Isrc/odt/src -I/usr/include 
>/qt4/phonon_compat -Ibin -Isrc -o bin/Carte.o src/Carte.cpp
>In file included from src/mainwindow.h:47:0,
>                 from src/Liaison.h:30,
>                from src/Carte.cpp:33:
>src/LecteurAudio.h:49:38: fatal error: Phonon/MediaSource: Aucun fichier ou dossier de ce type
>         #include <Phonon/MediaSource>
>                                      ^
>compilation terminated.
>Makefile:873: recipe for target 'bin/Carte.o' failed
>make: *** [bin/Carte.o] Error 1

easily solved with  
> qmake-qt4 -r rolisteam.pro DEFINES+=_PHONON_INCLUDE_QT_

I now have a working binary thanks a lot !

On a side note : 
so far compilation only worked on the 1.6.1 tarball the git clone has the following error : 
>g++ -c -m64 -pipe -O2 -w -D_REENTRANT -D_PHONON_INCLUDE_QT_ -DHAVE_SOUND >-DVERSION_MAJOR=1 -DVERSION_MIDDLE=7 -DVERSION_MINOR=0 -DQT_NO_DEBUG >-DQT_MULTIMEDIA_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB >-DQT_SHARED -I/usr/share/qt4/mkspecs/linux-g++-64 -I. -I/usr/include/qt4/QtCore -I/usr/include >/qt4/QtNetwork -I/usr/include/qt4/QtGui -I/usr/include/qt4/QtXml -I/usr/include/qt4/QtMultimedia -I/usr >/include/qt4 -I. -Isrc -Isrc/odt/format_OO_oasis_ -Isrc/odt/src -Isrc/audio -Ibin -Isrc -o bin/OOReader.o >src/odt/format_OO_oasis_/OOReader.cpp
>src/odt/format_OO_oasis_/OOReader.cpp: In member function ‘bool >OOReader::convertBlock(QTextCursor&, QDomElement, int)’:
>src/odt/format_OO_oasis_/OOReader.cpp:491:57: error: ‘class QString’ has no member named >‘toHtmlEscaped’
>             QString pretag = QString(child.nodeValue()).toHtmlEscaped();
 >                                                        ^
>Makefile:935: recipe for target 'bin/OOReader.o' failed
>make: *** [bin/OOReader.o] Error 1

not that problematic since I successfully complied the tarball,
Again, thanks for the help and happy hacking :)

**Comment by obiwankennedy** [comment]
If you compile a clone from git, then you can compile with Qt5. :-)

The port to Qt5 is done in git. The documentation only describe the process for stable version.

**Comment by ledshark24** [comment]
Hi,

I just successfully compiled the git code however I encountered a couple of difficulties that the [wiki](http://wiki.rolisteam.org/index.php/CompilationLinux) did not mention.

1. the git submodule for DiceParser is set with the ssh url that cause issues when cloning from https url for poor souls without ssh keys ;-) 
2. the packet list is inconstant with the system, here is what I needed to install : 
`apt-get install git qtbase5-dev-tools qt5-qmake libqt5network5 zlib1g-dev build-essential qtmultimedia5-dev libqt5core5a libqt5gui5 qttools5-dev-tools `

In hope that  helps improving the CompilationLinux page.

 Again, thanks for the impressive work and happy hacking :)
